{"name": "test_getting2_post", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "def test_getting2_post():\n        response = requests.get(url=SERVICE_URL + \"895\")\n>       assert response.status_code == 200\nE       assert 404 == 200\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_request1.py:31: AssertionError"}, "start": 1752052403973, "stop": 1752052404690, "uuid": "a5c2dc75-3aaf-45fc-95bc-6e57f4931312", "historyId": "35ae82baa869bedfbb2b4dc5860318c1", "testCaseId": "35ae82baa869bedfbb2b4dc5860318c1", "fullName": "tests.test_request1#test_getting2_post", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_request1"}, {"name": "host", "value": "LP-0731"}, {"name": "thread", "value": "24924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_request1"}]}